airflow/api_connexion/openapi/v1.yaml

```diff
@@ -5499,7 +5517,14 @@ components:
         items:
           type: string
       description: The value can be repeated to retrieve multiple matching values (OR condition).
-
+    FilterExecutor:
```

Comment:
Do we need an index for that one  (see discussion by @dstandish )

Reply:
Hey actually I feel even more confident about it generally avoiding indexes for rest api. Found there are more reasons for it being slow than just indexing.  So my position is NO!

I’ll send little more to dev list at some point